packages/core/src/common/keybinding.ts

```diff
@@ -113,9 +125,8 @@ export class KeybindingContextRegistry {
 @injectable()
 export class KeybindingRegistry {
 
+    private keymaps: Keybinding[][] = [[], [], [], []];
```

Comment:
Is there a better way to initialize this? I'd like for it to be bound by the amount of constants in the enum above, without having to do it in the constructor...

Reply:
keybinding[][keybindingscope.end] ?

Reply:
Still makes it so that this.keymaps[0] is undefined no?

Reply:
The other option I had is this in the constructor... 
```
        this.keymaps = [];
        Object.keys(KeybindingScopes).forEach(key => this.keymaps.push([]));
```

Reply:
That wont work since:

```
> Object.keys(KeybindingScopes)
[ '0', '1', '2', '3', 'DEFAULT', 'USER', 'WORKSPACE', 'END' ]
```

Better of using something like 
`for let i = 0; i < KeybingScopes.END; i++) { this.keymaps.push([])}`

Reply:
Oops you're right!